src/docs/product/performance/transaction-summary.mdx

```diff
@@ -53,6 +53,18 @@ When viewing transactions, you may want to create more curated views. Click "Ope
 
 _Note_: Currently, only transaction data - the transaction name and any attributes the transaction inherits from its root span - is searchable. Data contained in spans other than the root span is not indexed and therefore cannot be searched.
 
+## Suspect Tags
```

Comment:
Should we mention this is an EA feature?

Reply:
If it is, then, yes. Here's our standard text, Dora:
```suggestion
## Suspect Tags

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>
```